config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,27 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.dig('kafka', 'brokers')[0]
```

Comment:
Are we guaranteed a single broker? Why not stick with the `map` logic below?

Reply:
apps I've checked go by default with the first one (see [this](https://github.com/RedHatInsights/insights-ingress-go/blob/af9fff20de2cfd98a97b507fd5d088de5a907a9c/internal/config/config.go#L148))

Another point is , in Clowder, the authtype is tied to each broker, so each broker should have the same auth. mode in order for the config to work in every situation

Reply:
See https://consoledot.pages.redhat.com/clowder/dev/providers/kafka.html

you can see the auth. info is on each `broker` section, so we'll have to do handle that somehow ...suggestions are welcome

Reply:
Yeah but they come from Clowder on a per-broker  basis - it is not expected to have different brokers with different auth methods though, so we decided to default with the first broker auth. config